lib/documents/schemas/trademark_decisions.json

```diff
@@ -274,12 +274,12 @@
       "key": "trademark_decision_date",
       "name": "Decision date",
       "preposition": "Decision date",
+      "short_name": "Date",
```

Comment:
Nice. I'd be tempted to pull the reorder commits out into a separate PR, since they're not connected to the nested facets work and are something that provide value in isolation.